src/components/ValidateCodeActionModal/ValidateCodeForm/BaseValidateCodeForm.tsx

```diff
@@ -90,6 +91,10 @@ function BaseValidateCodeForm({
     // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing -- nullish coalescing doesn't achieve the same result in this case
     const shouldDisableResendValidateCode = !!isOffline || account?.isLoading;
     const focusTimeoutRef = useRef<NodeJS.Timeout | null>(null);
+    const [timeRemaining, setTimeRemaining] = useState(CONST.REQUEST_CODE_DELAY as number);
+    const [isResend, setIsResend] = useState(false);
```

Comment:
```suggestion
    const [isResent, setIsResent] = useState(false);
```

I think this is more descriptive